geowebcache/rest/src/main/java/org/geowebcache/rest/config/GWCRestConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.geowebcache.rest.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurationSupport;
+
+/**
+ * Configure various aspects of Spring MVC
+ */
+@Configuration
+@EnableWebMvc
+public class GWCRestConfig extends WebMvcConfigurationSupport {
+
+    @Autowired
+    private ApplicationContext applicationContext;
+
+}
```

Comment:
I don't think this class should be needed at all? If there is something that's needed from this, it would be best if it could be moved to geowebcache-rest-context.xml, otherwise it will interfere with GeoServer rest config.